### PR TITLE
Fix/workaround for character limits

### PR DESCRIPTION
**Pull request by pavlovcik**
Resolves https://github.com/ubiquibot/comment-incentives/issues/19

Had to set up jest to get this to run locally (mocks.) Doing sort of a two-for-one and setting up some basic but much needed tests.

- To be honest I'm not 100% sure if this is the root problem fix but @gitcoindev's [research seems to concur](https://github.com/ubiquibot/comment-incentives/issues/19#issuecomment-1948818100). 
- The hypothesis is another error dumps into the comment which overloads the limit. 
- I believe I fixed the error (explanation [here](https://github.com/ubiquibot/comment-incentives/issues/19#issuecomment-1948851259))

### CODE REVIEW

File: src/handlers/issue/assignee-scoring.ts
Comment (pavlovcik) [inline src/handlers/issue/assignee-scoring.ts:1]: Basically the real fixes are all in this file.

**Comment by molecula451** [comment]
code looks good, can you paste us some sort of debugging output/result ?

**Comment by pavlovcik** [comment]
> code looks good, can you paste us some sort of debugging output/result ?

[log.txt](https://github.com/ubiquibot/comment-incentives/files/14313918/log.txt)

**Comment by pavlovcik** [comment]
Let's give it a test @molecula451

**Comment by molecula451** [comment]
on my way!

**Pull request merged**